### ISSUE TITLE
type 54

### ISSUE THREAD

**Issue by pic** [simplificator/simplificator-withings#14]
@pascalbetz 

it seems we have a new 54 type coming in:
Unknown {"value"=>-1000, "type"=>18, "unit"=>-3}

Also, raising an exception for unknown type is maybe not the best option, what about a warning instead?

**Comment by pascalbetz** [comment]
@pic true, raising seems like a bad idea now. i'd just ignore unknown types.... 

btw neither type 54 nor type 18 are documented....

http://vitrine.withings.com/api-developer/documentation#category-values

**Comment by pascalbetz** [comment]
@pic: can you check if 0.6.10 is what you were looking for and close if OK?
